app/templates/admin/messages.hbs

```diff
@@ -1 +1,58 @@
-{{outlet}}
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui stackable very basic table compact">
+        <thead>
+          <tr>
+            <th>{{t 'Recipient'}}</th>
+            <th>{{t 'Trigger'}}</th>
+            <th>{{t 'Email Message'}}</th>
+            <th>{{t 'Notification Message'}}</th>
+            <th>{{t 'Options'}}</th>
+            <th>{{t 'Sent At'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |message|}}
+            <tr>
+              <td>
+                <div class="ui ordered list">
+                  {{#each message.recipient as |recipient|}}
+                    <div class="item">{{recipient.name}}</div>
+                  {{/each}}
+                </div>
+              </td>
+              <td>
+                {{message.trigger}}
+              </td>
+              <td>
+                <h5 class="ui header">{{message.emailMessage.subject}}</h5>
+                {{message.emailMessage.message}}
+              </td>
+              <td>
+                <h5 class="ui header">{{message.notificationMessage.subject}}</h5>
+                {{message.notificationMessage.message}}
+              </td>
+              <td>
+                <div class="ui ordered list">
```

Comment:
<img width="1266" alt="screen shot 2017-06-14 at 3 17 58 pm" src="https://user-images.githubusercontent.com/12716067/27126337-ba1cb822-5114-11e7-92f0-36fce3661950.png">
@geekyd For uniformity I have used ordered list, do you still want me to change it